zerver/tornado/event_queue.py

```diff
@@ -938,7 +938,7 @@ def process_message_event(
     message_type: str = wide_dict["type"]
     sending_client: str = wide_dict["client"]
 
-    @cachify
+    @lru_cache(maxsize=None)
```

Comment:
Can you comment on the differences, if any, between the behavior of `cachify` and `lru_cache(maxsize=None)` in the commit message?